cypress/plugins/testDataProcessor.js

```diff
@@ -108,9 +108,11 @@ function processSetResponseJson(setResponseJsonData) {
  *  getErrorContentObjectJson()
  */
```

Comment:
Fetching the error content json file from the fcs and config modules and then combining and storing it in env